Tests/GRPCNIOTransportCoreTests/Client/Connection/LoadBalancers/PickFirstLoadBalancerTests.swift

```diff
@@ -307,6 +308,7 @@ final class PickFirstLoadBalancerTests: XCTestCase {
           // Must only be connected to server 2 now.
           XCTAssertEqual(context.servers[0].server.clients.count, 0)
           XCTAssertEqual(context.servers[1].server.clients.count, 1)
+          XCTAssertNotNil(context.loadBalancer.pickSubchannel())
```

Comment:
Without the fix this returns `nil` even though a connection is ready and available to use.